tests/system/providers/google/cloud/dataplex/example_dataplex_dq.py

```diff
@@ -0,0 +1,308 @@
+    # [START howto_dataplex_data_scan_job_state_sensor]
+    get_data_scan_job_status = DataplexDataQualityJobStatusSensor(
+        task_id="get_data_scan_job_status",
+        project_id=PROJECT_ID,
+        region=REGION,
+        data_scan_id=DATA_SCAN_ID,
+        job_id=str(job_id),
+    )
+    # [END howto_dataplex_data_scan_job_state_sensor]
+    # [START howto_dataplex_get_data_quality_job_operator]
+    get_data_scan_job_result = DataplexGetDataQualityScanResultOperator(
+        task_id="get_data_scan_job_result",
+        project_id=PROJECT_ID,
+        region=REGION,
+        data_scan_id=DATA_SCAN_ID,
+    )
+    # [END howto_dataplex_get_data_quality_job_operator]
+    # [START howto_dataplex_get_data_quality_job_def_operator]
+    get_data_scan_job_result_def = DataplexGetDataQualityScanResultOperator(
+        task_id="get_data_scan_job_result_def",
+        project_id=PROJECT_ID,
+        region=REGION,
+        data_scan_id=DATA_SCAN_ID,
+        deferrable=True,
+    )
+    # [END howto_dataplex_get_data_quality_job_def_operator]
+    # [START howto_dataplex_delete_asset_operator]
+    delete_asset = DataplexDeleteAssetOperator(
+        task_id="delete_asset",
+        project_id=PROJECT_ID,
+        region=REGION,
+        lake_id=LAKE_ID,
+        zone_id=ZONE_ID,
+        asset_id=ASSET_ID,
+    )
+    # [END howto_dataplex_delete_asset_operator]
+    delete_asset.trigger_rule = TriggerRule.ALL_DONE
+    # [START howto_dataplex_delete_zone_operator]
+    delete_zone = DataplexDeleteZoneOperator(
+        task_id="delete_zone",
+        project_id=PROJECT_ID,
+        region=REGION,
+        lake_id=LAKE_ID,
+        zone_id=ZONE_ID,
+    )
+    # [END howto_dataplex_delete_zone_operator]
+    delete_zone.trigger_rule = TriggerRule.ALL_DONE
+    # [START howto_dataplex_delete_data_quality_operator]
+    delete_data_scan = DataplexDeleteDataQualityOperator(
+        task_id="delete_data_scan",
+        project_id=PROJECT_ID,
+        region=REGION,
+        data_scan_id=DATA_SCAN_ID,
+    )
+    # [END howto_dataplex_delete_data_quality_operator]
+    delete_data_scan.trigger_rule = TriggerRule.ALL_DONE
+    delete_lake = DataplexDeleteLakeOperator(
+        project_id=PROJECT_ID,
+        region=REGION,
+        lake_id=LAKE_ID,
+        task_id="delete_lake",
+        trigger_rule=TriggerRule.ALL_DONE,
+    )
+
+    chain(
+        # TEST SETUP
+        create_dataset
+        >> [create_table_1, create_table_2]
+        >> insert_query_job
+        >> create_lake
+        >> create_zone
+        >> create_asset
+        # TEST BODY
+        >> create_data_scan
+        >> execute_data_scan
+        >> execute_data_scan_def
+        >> run_data_scan
+        >> get_data_scan_job_status
+        >> get_data_scan_job_result
+        >> get_data_scan_job_result_def
+        # TEST TEARDOWN
+        >> delete_asset
+        >> delete_zone
+        >> delete_data_scan
+        >> delete_lake
+    )
```

Comment:
I can see that there is a step for creating 2 tables, but no step with deleting them

Reply:
done, the dataset with the tables is now deleted